app/pubsub/data.go

```diff
@@ -60,7 +61,7 @@ func (s *SubscriptionManager) Subscribe(req *SubscriptionRequest) {
 		case "transaction":
 			s.Consumers[req.Topic()] = NewTransactionConsumer(s.Client, tmp, s.Connection, s.DB, s.ConnLock, s.TopicLock, s.Counter)
```

Comment:
How come we're not having kafkaWrite for `transaction` type too? We should be writing transactions and events to kafka.